### DISCUSSION TITLE
Vercel deployment err - failed to connect to postgres: failed to connect to `host=aws-0-us-east-1.pooler.supabase.com user=postgres.user.database=postgres`: failed SASL auth (invalid SCRAM server-final-message received from server)

### DISCUSSION THREAD

**Discussion by JJRobertsSex30** [mckaywrigley/chatbot-ui#1766] [Help]
Trying to deploy this on vercel

First question, as part of this process it asks for the db password but says you can leave it blank if you want. Am I supposed to create a db password here or enter the existing password? Because where is the existing password if it is required?

Anyway I left it blank and when trying to push the db I got this error ...

failed to connect to postgres: failed to connect to `host=aws-0-us-east-1.pooler.supabase.com user=postgres.user.database=postgres`: failed SASL auth (invalid SCRAM server-final-message received from server)

**Comment by Davitcoin** [comment]
Supabase gives you a master password once you create you´re account, you need to keep it cause that is the password you´re going to use for setting up DB´s, try to recover your password and replace "[Your-psswd]" postgresql://postgres.example:[YOUR-PASSWORD]@aws-0-us-west-1.pooler.supabase.com:6543/postgres
